src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -1121,5 +1128,23 @@ bool PerformInteractionRequest::SetChoiceIdToResponseMsgParams(
   return true;
 }
 
+bool PerformInteractionRequest::IsAllChoiceSetIdsAllowed(
+    ApplicationSharedPtr app,
+    const ns_smart_device_link::ns_smart_objects::SmartObject&
+        choice_set_id_list) const {
+  SDL_LOG_AUTO_TRACE();
+  const size_t choice_set_id_list_length = choice_set_id_list.length();
```

Comment:
@ychernysheva you can convert this list to vector using `asArray()` method and then `std::any_of` + lambda for return result

Reply:
But in this case we will have unary predicate inside of std::any_of and another lambda outside of this function - for me this solution seems to be too overloaded and is harder to read than proposed implementation with loop

Reply:
Done in a90a2bb